sql/rpl_info_table.cc

```diff
@@ -21,6 +21,46 @@
 #include "rpl_info_values.h"        // Rpl_info_values
```

Comment:
I am not sure, Laurynas, because that test just measures the amount of allocated memory before some action and after it. And if the difference between two measures is big enough then the test failed. But the question is how to understand that the difference is big enough? Where is the guarantee that delta is abnormal?